dev_guide/managing_images.adoc

```diff
@@ -350,6 +350,47 @@ metadata:
 ----
 ====
 
+[[using-is-with-k8s]]
+== Using Image Streams with Kubernetes resources (Technology Preview)
+Image Streams, being {product-title} native resources, work out of the box with
+all the rest of native resources available in {product-title}, such as
+xref:../dev_guide/builds/index.adoc#dev-guide-how-builds-work[Builds] or
+xref:../dev_guide/deployments/basic_deployment_operations.adoc#dev-guide-basic-deployment-operations[Deployments].
+Currently, it is also possible to make them work with native Kubernetes resources,
+such as xref:../../dev_guide/jobs.adoc#dev-guide-jobs[Jobs],
+xref:../architecture/core_concepts/deployments.adoc#replication-controllers[Replication Controllers],
+xref:../architecture/core_concepts/deployments.adoc#replication-controllers[Replication Controllers] or
```

Comment:
Yes, I've meant Replica Sets here, I've opened https://github.com/openshift/openshift-docs/pull/4935 to add RS there.